src/main/java/ttt/Board.java

```diff
@@ -99,52 +59,51 @@ public boolean isFinished() {
         return (isDraw() || isWon());
```

Comment:
I think Felipe already mentioned that you don't need the outer brackets here. You could say `return isDraw() || isWon();`